specification/appconfiguration/AppConfiguration/main.tsp

```diff
@@ -33,6 +37,15 @@ namespace AzureAppConfiguration;
 @doc("Service API versions")
 enum Versions {
   @useDependency(Azure.Core.Versions.v1_0_Preview_2)
-  @doc("The 2023-11-01 API version")
```

Comment:
Do we need to keep the older API version?

Reply:
If we keep the older API version, the SAS token auth will be added to older API version as well.